frida-trace-tools/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "frida-trace",
```

Comment:
This clashes with the parent package name.

Reply:
Oops, renamed to `frida-trace-tools` in latest